staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -226,10 +226,19 @@ type GroupResources struct {
 	// The empty string represents the core API group.
 	// +optional
 	Group string
-	// Resources is a list of resources within the API group. Subresources are
-	// matched using a "/" to indicate the subresource. For example, "pods/log"
-	// would match request to the log subresource of pods. The top level resource
-	// does not match subresources, "pods" doesn't match "pods/log".
+	// Resources is a list of resources this rule applies to.
+	//
+	// For example:
+	// 'pods' means pods.
```

Comment:
nit: Use "matches" instead of "means"?

- 'pods' matches requests to pods
- 'pods/logs' matches requests to the logs subresource of pods
- '*' matches requests to all resources
- 'pods/*' matches requests to all subresources of pods
- '*/scale' matches requests to the scale subresource of any resource

Reply:
ok